crates/oxc_language_server/src/snapshots/fixtures_linter_multiple_suggestion@forward_ref.ts.snap

```diff
@@ -0,0 +1,4 @@
+---
+source: crates/oxc_language_server/src/tester.rs
+---
+No diagnostic reports
```

Comment:
Hm... I expect at least a diagnostic. Extra changed the rule to disable the Framework flags to be sure it always run:

```
    fn should_run(&self, ctx: &ContextHost) -> bool {
        true // ctx.frameworks().contains(FrameworkFlags::React)
    }
```

Reply:
nvm should check if I put the right path 😓